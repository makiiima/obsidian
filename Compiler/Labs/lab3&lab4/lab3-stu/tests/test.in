3+4*5
-(3+4)*5
10/3*3
3+(1/2)
-10+2^3+9/3-5
