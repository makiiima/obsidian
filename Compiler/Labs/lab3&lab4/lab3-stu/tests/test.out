35
-35
9
